pkg/version/version.go

```diff
@@ -0,0 +1,103 @@
+package version
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/metadata"
+	javadb "github.com/aquasecurity/trivy-java-db/pkg/db"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/policy"
+)
+
+var (
+	Version = "dev"
```

Comment:
I want to make it read-only.

```suggestion
	version = "dev"
```

Then, can we add `Version() string` or `AppVersion() string`?